### PR TITLE
Ch-ch-ch-ch-changes

### PR DESCRIPTION
**Pull request by ajaypillarisetti**
A set of changes per the issues / past weeks discussion. Biggest changes to the dataset view and stylesheet. Edited config to point to Maroon; edited models to include a "training" column in the DataPoints table.

### CODE REVIEW

**Comment by jeremyrcoyle** [comment]
This is awesome!

File: models.py [hunk 1]
```diff
@@ -70,6 +69,17 @@ def prev(self):
       .order_by(Datasets.created_at)\
       .first()
 
+  def items(self):
+    return Datasets.query\
+      .filter(Datasets.study_id==self.study_id)\
+      .order_by(Datasets.created_at)\
+      .all()
+
+  def study_name(self):
```
Comment (marcpare) [inline models.py]: The `study_name` method isn't needed.

The Datasets model already has a `study` property as a backref from Studies

```
datasets = db.relationship('Datasets', lazy="dynamic", backref="study")
```

So all you need to do is...

```
dataset.study.name
```

to get the name. The ORM generates the query for you.

**Comment by marcpare** [comment]
Cool to see changes to this! Like the new style. Hope these tips help.

**Comment by ajaypillarisetti** [comment]
Super helpful. More coming soon. Thanks much Marc!

**Comment by ajaypillarisetti** [comment]
Some more cosmetic changes and some changes to the database to note when a user clicks "Done" on the dataset page. 

None of the underlying logic is in place, though: the done button needs to trigger a change in that table and then display the check based on the value of that column. Currently just showing checks for everything.

We also need to discuss a bit more how we're going to deal with multiple labelers working on the same dataset (does the user who creates the database invite others? what database structure makes most sense?).

**Pull request merged**